HVSample/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
     }
+    packagingOptions {
+        exclude 'META-INF/LICENSE'
```

Comment:
Talked to Justin (PM).  I have added task # 55495
 to ensure we do the right thing

---
In reply to: [116609603](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r116609603) [](ancestors = 116609603)